### PR TITLE
Protects heads from antagonist roles as a config option and enables it on MRP only.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Protects head roles from antagonist as a config option, and enables it on MRP, but not LRP (subject to change.)

## Why It's Good For The Game

Antagonist heads makes less sense in an RP perspective, since they should be someone you can follow orders from as a new player without being given false orders killing everyone and getting bwoinked when you try it again.

## Changelog
:cl:
config: Head roles on MRP are protected from antagonist roles
/:cl:

### CODE REVIEW

**Comment by TeomanTheGreat** [comment]
You do understand now there will be little to no paranoia in command  right? This game isnt just chilling at the [INSERT DEPARTMENT NAME HERE] simulator.

**Comment by Archanial** [comment]
Only hop should be protected if there is no captain.

**Comment by TeomanTheGreat** [comment]
Make it so hop, cap, and hos are the only ones protected

**Comment by PowerfulBacon** [comment]
> 
> 
> You do understand now there will be little to no paranoia in command right? This game isnt just chilling at the [INSERT DEPARTMENT NAME HERE] simulator.

Heads can still be converted to gangs, can still be absorbed by changelings and converted by other conversion antags. They just will not be roundstart antags on MRP, since its very hard to listen to your command staff when they are giving faulty orders out. Being traitor head of staff is a lot of fun because of how powerful you are, but it doesn't work well in a roleplay environment where people are exected to listen to their superiors and ask them for help.

**Comment by Archanial** [comment]
> 
> 
> > You do understand now there will be little to no paranoia in command right? This game isnt just chilling at the [INSERT DEPARTMENT NAME HERE] simulator.
> 
> Heads can still be converted to gangs, can still be absorbed by changelings and converted by other conversion antags. They just will not be roundstart antags on MRP, since its very hard to listen to your command staff when they are giving faulty orders out. Being traitor head of staff is a lot of fun because of how powerful you are, but it doesn't work well in a roleplay environment where people are exected to listen to their superiors and ask them for help.

By your own logic malf AI should be disabled because borgs have listen to its faulty orders.

> Antagonist heads makes less sense in an RP perspective

this argument is hilariously bad. Why can't you RP a rogue RD, CMO or whoever?

> since they should be someone you can follow orders from as a new player without being given false orders killing everyone and getting bwoinked when you try it again

No. getting bwoinked is part of this games experience. You learn but you also make mistakes. If you aren't sure what can and can't you do ahelp, ask mentors or just read the rules and use your common sense.

In conclusion, stop making mrp less fun "because rp reason",

**Comment by PowerfulBacon** [comment]
> By your own logic malf AI should be disabled because borgs have listen to its faulty orders.

The difference with malf AI is the borgs are antags too. When a head roles traitor the entire department doesn't become traitors too.

> In conclusion, stop making mrp less fun "because rp reason",

This won't ruin MRP, making the game completely unplayable because heads will no longer be antagonist, it will smoothen out the experience for new players and hopefully allow for some new roleplay expectations making heads of staff not just a job role with some extra gear.

> No. getting bwoinked is part of this games experience.

It's always nicer for new players to not be bwoinked, and its a part of the game's experience that most people should want to avoid

File: code/game/gamemodes/brother/traitor_bro.dm [hunk 1]
```diff
@@ -24,6 +24,8 @@
 		restricted_jobs += protected_jobs
 	if(CONFIG_GET(flag/protect_assistant_from_antagonist))
 		restricted_jobs += "Assistant"
+	if(CONFIG_GET(flag/protect_heads_from_antagonist))
```
Comment (qwertyquerty) [inline code/game/gamemodes/brother/traitor_bro.dm:27]: Does this really have to be done with each individual game mode?

**Comment by Archanial** [comment]
>  it will smoothen out the experience for new players

This is only your opinion.

File: code/game/gamemodes/brother/traitor_bro.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/game/gamemodes/brother/traitor_bro.dm:27]: Yea, like the other 2 flags

**Review by ike709 — APPROVED** [review]
Approved by headmin decree and me not disagreeing with said decree.

**Comment by ike709** [comment]
@Crossedfall Config.

**Review by Crossedfall — APPROVED** [review]
Config applied

**Pull request merged**